File: Selenium/Activities/Activity2.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {
    public static void main(String[] args){
        WebDriver driver =  new FirefoxDriver();
        driver.get("https://www.training-support.net");

        String title = driver.getTitle();
        System.out.println(title);

        WebElement idText= driver.findElement(By.id("about-link"));
        System.out.println("Text in element:"+ idText.getText());

        WebElement classText = driver.findElement(By.className("green"));
        System.out.println("Text is:" + classText.getText());

        WebElement linkValue = driver.findElement(By.linkText("About Us"));
        System.out.println("Text is:" + linkValue.getText());

        WebElement cssValue = driver.findElement(By.cssSelector(".green"));
        System.out.println("Text is: "+ cssValue.getText());

        driver.close();
    }
}

File: Python/Activities/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    #navigate to browser
    driver.get("https://www.training-support.net")

print("Title of the page is: {driver.title}")
about_us_link= driver.find_element(By.ID,"about-link")
about_us_link= driver.find_element(By.CLASS_NAME,"green")
about_us_link= driver.find_element(By.LINK_TEXT,"About Us")
about_us_link= driver.find_element(By.CSS_SELECTOR,"green")
print(about_us_link.text)
about_us_link.click()
driver.quit()
